locales/en/fields.json

```diff
@@ -9,7 +9,7 @@
     },
     "prison_number": {
       "label": "Prison number",
-      "hint": "For example, 'A1234AB'"
+      "hint": "For example, ‘A1234AB’"
```

Comment:
Which sort of quotes should be used inside a string?

Reply:
*reads the description*

Ah! Do we do this everywhere else?

Reply:
Yes, it's something I've always tried to do. Bit of a personal quest after a content designer I worked with pointed out the difference to me. But I don't think we have any form of linting to enforce it so it's usually just something I look out for in PR reviews. 

GOV.UK always tries to do it well and you'll see examples in some of [their guides](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#single-quotes) too. A common one I always try and make sure is correct is to use `don’t` (an apostrophe) instead of `don't` (a prime).